// mssql-i/es/model/SchemaModel.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var TableSchemaModel = (function () {
    function TableSchemaModel() {
    }
    return TableSchemaModel;
}());
exports.TableSchemaModel = TableSchemaModel;
var ColumnSchemaModel = (function () {
    function ColumnSchemaModel() {
    }
    return ColumnSchemaModel;
}());
exports.ColumnSchemaModel = ColumnSchemaModel;
var ProcedureSchemaModel = (function () {
    function ProcedureSchemaModel() {
    }
    return ProcedureSchemaModel;
}());
exports.ProcedureSchemaModel = ProcedureSchemaModel;
var ProcedureParSchemaModel = (function () {
    function ProcedureParSchemaModel() {
    }
    return ProcedureParSchemaModel;
}());
exports.ProcedureParSchemaModel = ProcedureParSchemaModel;
var SchemaModel = (function () {
    function SchemaModel() {
    }
    SchemaModel.prototype.getTableSchemaModel = function (tableName) {
        return this.tables.filter(function (table) { return table.name === tableName; })[0];
    };
    SchemaModel.prototype.getProcedureSchemaModel = function (name) {
        return this.procedures.filter(function (procedure) { return (procedure.name === name); })[0];
    };
    return SchemaModel;
}());
exports.SchemaModel = SchemaModel;
